runtime/dart_vm.cc

```diff
@@ -521,6 +521,8 @@ DartVM::~DartVM() {
     Dart_ExitIsolate();
   }
 
+  platform_isolate_manager_.ShutdownPlatformIsolates();
```

Comment:
I would expect shutdown of platform isolates to happen during shutdown of a Flutter engine instance (`Engine`/`Shell`/`RuntimeController`).

`DartVM` is a process-wide object that will remain alive after engine shutdown if at least one other engine instance is still active.  Or in some configurations the `DartVM` may stay alive for the entire process lifetime (see `Settings::leak_vm`)

Reply:
Done, but I'm not sure how to access the `RuntimeController` from `DartIsolate::OnShutdownCallback()` on the platform isolate. `platform_configuration()` is only non-null on the root isolate.